docs/KnowledgeBase/MapViewers/README.md

```diff
@@ -0,0 +1,22 @@
+---
+id: map-viewers-overview
+slug: map-viewers-overview
+title: Map viewers
+sidebar_position: 1
+tags:
+ - map-viewers
+ - viewers
+ - overview
+---
+
+There are so many map viewers out there, how do you choose the best one for your needs?
+Here we attempt to help you make this desicion.
+
+Disclaimer:
+Everything written in this knowledge base is based on our research of selected well known map viewers. We do not attempt to recommend one as the best, but give you all the information we believe you need in order to make a better desicion based on your use case.
+Some of the viewers may be incompatible with some of our data or services and will require additional adjustments in order to be consume them (for some it may not be possible at all).
```

Comment:
```suggestion
Some of the viewers may be incompatible with some of our data or services and will require additional adjustments in order to consume them (for some it may not be possible at all).
```